src/exasolvs/RequestDispatcher.lua

```diff
@@ -102,6 +112,17 @@ function RequestDispatcher:_init_logging(properties)
     end
 end
 
+local function xpcall_workaround(callback, error_handler, ...)
+    local probe <const> = "PROBE:error"
+    local _, actual = pcall(function() error(probe, 0) end)
+    if(actual == probe) then
```

Comment:
Extract this into a function `xpcall_workaround_required()` to improve readability